### PR TITLE
Update preliminary freight economy report availability status

### PR DESCRIPTION
**Pull request by mbrameshkumar**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/jQbqx9HR](https://trello.com/c/jQbqx9HR)

File: content/api/reports/_index.html [hunk 1]
```diff
@@ -12,10 +12,10 @@
 ---
 
 <div class="message--warn maxw48r pam mbl">
-  <strong>The Preliminary freight calculation report for Parcel Norway customers will be disabled from 21.06.2021</strong>
+  <strong>The Preliminary freight calculation report for Parcel Norway customers were disabled from 21.06.2021</strong>
```
Comment (larsaane) [inline content/api/reports/_index.html:15]: was disabled....

Comment (mbrameshkumar) [reply content/api/reports/_index.html:15]: Parcel Norway customers is plural hence i added as `were`..may be I will make it to Parcel Norway customer was disabled from June 21st. What do you think?

Comment (larsaane) [reply content/api/reports/_index.html]: Of course you are right :)

Comment (larsaane) [reply content/api/reports/_index.html]: Rewrite to customer was... so it is the same for all the text you changed.

**Review by larsaane — APPROVED** [review]
Good

**Pull request merged**